src/js/components/projects/detail.tsx

```diff
@@ -161,7 +167,7 @@ const ProjectDetail = (
     return <Redirect to={routes.project_detail(project.slug)} />;
   }
 
-  const hasEpics = epics && epics.epics.length > 0;
+  let viewTaskTab = false;
```

Comment:
This should probably be part of the reactive state (via `useState`). Additionally, for this to have any effect I think we'd need to pass it through as an argument to `useFetchTasksByProject`, and in that hook make the fetching conditional on `if (taskTabViewed) { ... }` or similar.